Parallel processing enabled
Logging model files to ./chkp/debug-test
TRAINING MODEL: regressor
Training on GPU: 4 devices
************* EPOCH: 1 *************
  0%|                                                                                                                      | 0/2445 [00:01<?, ?it/s]
Traceback (most recent call last):
  File "train/train_density_neonet.py", line 116, in <module>
    main(args)
  File "train/train_density_neonet.py", line 104, in main
    trainer.fit()
  File "/home/stephen/Projects/Learning-Frustum-Proposals/trainers/base_trainer.py", line 333, in fit
    self._get_wandb_plot(
  File "/home/stephen/Projects/Learning-Frustum-Proposals/trainers/density_trainer.py", line 304, in _get_wandb_plot
    self.train_dataloader.dataset.get_wandb_plot(
  File "/home/stephen/Projects/Learning-Frustum-Proposals/datasets/density_point_cloud_data.py", line 146, in get_wandb_plot
    feats_rgb = self.get_pc_feats(feats)
  File "/home/stephen/Projects/Learning-Frustum-Proposals/datasets/density_point_cloud_data.py", line 79, in get_pc_feats
    color = np.random.randint(0, 255, size=3).reshape(1, 3)
NameError: name 'np' is not defined